### PR TITLE
Release v4.23.2

### PR DESCRIPTION
**Pull request by FhanOmise**
#### 👾 Bug Fixes
- Handle capabilities failed. (PR [#294](https://github.com/omise/omise-woocommerce/pull/294))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+### [v4.23.2 _(Aug 8, 2022)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.23.2)
```
Comment (som-m) [inline CHANGELOG.md:3]:
```suggestion
### [v4.23.2 _(Aug 18, 2022)_](https://github.com/omise/omise-woocommerce/releases/tag/v4.23.2)
```

Comment (aashishgurung) [inline CHANGELOG.md:3]: Please change the date to the current date.

Comment (FhanOmise) [reply CHANGELOG.md]: Fixed here ka [3955645](https://github.com/omise/omise-woocommerce/pull/295/commits/395564518219f191ba128e79772ef728369bcc62)

**Review by aashishgurung — APPROVED** [review]
LGTM

**Review by new4762 — APPROVED** [review]

**Review by som-m — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

**Pull request merged**